### ISSUE TITLE
Favicon not appearing in 3.0.10

### ISSUE THREAD

**Issue by kanarb** [knice/ucsc-drupal#31]
I see the default drupal icon instead of the ucsc favicon. Tested in Firefox and Chrome on Mac.

**Comment by knice** [comment]
It has to be enabled in your theme settings by checking the "shortcut icon" 

![screen shot 2015-08-04 at 2 47 44 pm](https://cloud.githubusercontent.com/assets/36719/9073694/f1a619c0-3ab7-11e5-86bf-72924f3553a6.png)

**Comment by kanarb** [comment]
It was checked. I unchecked and rechecked and cleared the cache. The drupal
icon is gone, but the ucsc favicon still does not appear. Is it showing up
for you?
- B

**Comment by kanarb** [comment]
Actually I was wrong the drupal icon is still showing up.

**Comment by knice** [comment]
I'm seeing the favicon.

### Here are my theme settings:

![screen shot 2015-08-05 at 12 34 20 pm](https://cloud.githubusercontent.com/assets/36719/9095825/5531c744-3b6e-11e5-8be2-7c838677f887.png)

### And here is what shows up in my HTML source with those settings:

![napkin 2 08-05-15 12 37 12 pm](https://cloud.githubusercontent.com/assets/36719/9095900/c694bc66-3b6e-11e5-999a-38509cc930bc.png)

**Comment by knice** [comment]
I see what is happening. I uploaded the favicon through the Drupal interface and it was installed in `/sites/default/files/favicon.ico`.

**Comment by knice** [comment]
Ohhhhhhh, I figured it out. Check this setting:

![screen shot 2015-08-05 at 12 46 17 pm](https://cloud.githubusercontent.com/assets/36719/9096108/0735f48c-3b70-11e5-8982-760159b1f6fa.png)

I thought that meant "use the default _Drupal_ favicon." It actually means "use the favicon that came with _this theme_." Check that box and it works. The HTML source has the path to the favicon in the theme folder.

Closing this.
